tst/standard/attr.tst

```diff
@@ -1548,6 +1548,116 @@ gap> D := Digraph(OutNeighbours(CycleDigraph(13)));;
 gap> ChromaticNumber(D);
 3
 
+#  Test ChromaticNumber Lawler
+gap> ChromaticNumber(Digraph([[1]]) : lawler);
+Error, the argument <D> must be a digraph with no loops,
```

Comment:
Okay @EwanGilligan this is bonkers but I think (even though this should obviously be completely harmless!) that if you just delete this test, as well as the corresponding one for `Byskov`), then this should pass. (And those tests aren't even necessary for this PR, since that code is already tested).
```suggestion
```
I need to investigate why, but I think that's obviously independent of this PR...

Reply:
That's incredibly strange, but I'll give that a go